templates/partials/answers/checkbox.html

```diff
@@ -2,13 +2,21 @@
 
 {% if answer.label %}
   {% set checkbox_label = answer.label %}
-{% elif answer.options|length == 1 %}
-  {% set checkbox_label = '' %}
-{% else %}
+{% elif answer.no_label != True and answer.options|length != 1 %}
```

Comment:
The double negative makes this tricky to read, and there is now opportunity to fix the issue with answer.options|length == 1 also creating an empty element in the HTML. Something like 

```
{% if answer.no_label or answer.options|length == 1%}
   {{ onsCheckboxes({
    "legend": question_title,
    "id": answer.id,
    "legendClasses": "u-vh",
    "checkboxes": map_checkbox_config(form, answer),
    "mutuallyExclusive": mutuallyExclusive,
    "error": error
  }) }}

{% else %}

  {{ onsCheckboxes({
  "legend": question_title,
  "id": answer.id,
  "legendClasses": "u-vh",
  "checkboxesLabel": answer.label or _("Select all that apply"),
  "checkboxes": map_checkbox_config(form, answer),
  "mutuallyExclusive": mutuallyExclusive,
  "error": error
  }) }}

{% endif %}
```

Reply:
Makes it much shorter, done!